prisma/schema.prisma

```diff
@@ -63,21 +63,21 @@ model MenuItem{
   description String
   price Decimal @db.Decimal(12,2)
   restaurantId String 
-  imageId Int @unique
+  imageId String
   restaurant Restaurant @relation(fields: [restaurantId], references: [id])
   modificationGroups ModificationGroup[]   
   orderLines OrderLine[]
-  image MenuItemImage @relation(fields: [imageId], references: [id])
+  image Image @relation(fields: [imageId], references: [id])
 }
 
-model MenuItemImage{
-  id Int @id @default(autoincrement())
+model Image{
+  id String @id @default(uuid())
```

Comment:
Wrong type here. Migration creates TEXT field on db, this is not good. I think we are missing something here